python/libucx/setup.py

```diff
@@ -22,13 +22,17 @@ def run(self):
         super().run()
 
         with open("VERSION") as f:
-            version = f.read().strip()
+            package_version = f.read().strip()
+        
+        # strip off any other non-UCX version components, like ".post1"
+        major, minor, patch, *_ = package_version.split(".")
+        ucx_tag = f"v{major}.{minor}.{patch}"
```

Comment:
Let's use the packaging library instead since that's the canonical implementation. This should work (with the appropriate added import; feel free to reshuffle the code however of course):
```suggestion
        # strip off any other non-UCX version components, like ".post1"        
        ucx_tag = f"v{packaging.version.parse(package_version).base_version}"
```

Reply:
Ok sure, added in https://github.com/rapidsai/ucx-wheels/pull/5/commits/7f57c71ea1c10d8c2583898efad90ed0fb7788aa.